docs/manual/download.md

```diff
@@ -14,16 +16,24 @@ The C# library is distributed as two separate packages for Windows Desktop and W
 > [!NOTE]
 > As per existing C# NuGet packages convention, and unlike the C++ library below, the Desktop package has no suffix, and the UWP package adds a `.UWP` suffix.
 
-The C# library packages contain the C# assembly `Microsoft.MixedReality.WebRTC` as well as the per-architecture native DLLs. Therefore those packages are standalone, and there is no need to also reference the C++ library packages in your project.
+The C# library packages contain the C# assembly `Microsoft.MixedReality.WebRTC` as well as the per-architecture native DLLs. Therefore those packages are standalone, and there is no need to reference any other package in your project.
 
 ## C++ library
 
-The C++ library is distributed as two separate packages for Windows Desktop and Windows UWP:
+The C++ library architecture and distribution changed between the 1.0 and 2.0 releases.
+
+### New standalone library (v2.0)
+
+Starting from v2.0, the C++ library is a standalone source-only library building on top of the native implementation module `mrwebrtc.dll`. The legacy `Microsoft.MixedReality.WebRTC.Native.dll` previously containing the native implementation is gone. There is currently no precompiled binaries of that `mrwebrtc.dll` implementation module. The C++ library itself is only distributed as sources for C++ projects to integrate it and statically link with it.

```

Comment:
Consumers of 2.0 shouldn't need to care about a `Microsoft.MixedReality.WebRTC.Native.dll` that previously contained a native implementation and now is gone.

Also 
```
There is currently no precompiled binaries
```
True, but see other comment - this phrasing means that as soon as we have new packages we need to come back here (and to all the "transitional" doc blocks) and change this. I'd say we don't need to specify, for now if the docs say that there is a mrwebrtc.dll in 2.0 packages they are not lying! And by the time we make a release with this docs there are going to be packages by definition 😄

Reply:
Fixed.

Reply:
I was only thinking of removing those two sentences, but yeah after those are gone there's not a lot of meat left in these paragraphs. Ship it!